script/openqa-verify-pr

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+# Copyright 2024 (c) SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Schedule a verification run for a os-autoinst PR
+# USAGE:
+#
+# schedule_verification_run $SOURCE $PR_ID $BASE_TEST_ID $[test/module, test/module, ...] $TEST_NAME
+#
+# Parameters
+#
+# * $SOURCE - The source you want to schedule a rund for. Can be either opensuse or suse.
+# * $PR_ID - The ID of your GitHub Pull Request.
+# * $BASE_TEST_ID - The ID of the test you want you use as a base.
+# * $[test/module, ...] - The modules of your test you want to schedule for.
+# * $TEST_NAME - The Name of your test. (E.g.: TETS=RUST)
+#
+# Specify your GitHub access token in a config file at '~/.config/openqa/gh.conf' like this:
+#
+# ```
+# GH_ACCESS_TOKEN="YOUR_TOKEN_HERE"
+# ```
+#
+# Default test modules
+#
+# The following modules are automatically selected to load for your run:
+#
+# * 'tests/installation/bootloader_start'
+# * 'tests/boot/boot_to_desktop'
+# 
+# Example
+#
+# Pass arguments to this script like this:
+#
+# schedule_verification_run opensuse TOKEN 1234 12345 tests/console/rustup,tests/console/cargo RUST
+
+
+help_text=$(cat <<'EOF'
+OpenQA verification run scheduler.
+
+Usage:
+ schedule_verification_run $SOURCE $PR_ID $BASE_TEST_ID $[test/module, test/module,...] $TEST_NAME
+
+Parameters:
+ SOURCE - The source you want to schedule a run for. Can be either opensuse or suse.
+ PR_ID - The ID of your GitHub Pull Request.
```

Comment:
```suggestion
 PR_ID - The ID of your GitHub pull request.
```
(in accordance with https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests)

Reply:
fixed 👍🏼